contents/node/nodejs.md

```diff
@@ -0,0 +1,224 @@
+# Introduction to Node.js
+
+Author: Rachael Sim
+
+# Overview
+
+*Node.js* is a "JavaScript runtime built on Chrome’s V8 JavaScript engine. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient."
```

Comment:
The quotation seems a bit off. Maybe you could do without it?

*Node.js* is a JavaScript runtime built on Chrome’s V8 JavaScript engine. It uses an event-driven, non-blocking I/O model that makes it lightweight and efficient.